src/main/java/mate/academy/spring/service/impl/AuthenticationServiceImpl.java

```diff
@@ -10,20 +12,36 @@
 public class AuthenticationServiceImpl implements AuthenticationService {
     private final UserService userService;
     private final ShoppingCartService shoppingCartService;
+    private final RoleService roleService;
 
     public AuthenticationServiceImpl(UserService userService,
-                                     ShoppingCartService shoppingCartService) {
+                                     ShoppingCartService shoppingCartService,
+                                     RoleService roleService) {
         this.userService = userService;
         this.shoppingCartService = shoppingCartService;
+        this.roleService = roleService;
     }
 
     @Override
-    public User register(String email, String password) {
+    public User registerUser(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
+        user.setRoles(Set.of(roleService.getRoleByName("USER")));
         userService.add(user);
         shoppingCartService.registerNewShoppingCart(user);
         return user;
     }
```

Comment:
registerUser and Register admin looks like equal methods, lets better create one method reguster and get role as incoming parameter